consent/types.go

```diff
@@ -166,79 +166,47 @@ type RequestWasHandledResponse struct {
 // swagger:model acceptConsentRequest
 type HandledConsentRequest struct {
 	// ID instead of Challenge because of pop
-	ID string `json:"-" db:"challenge"`
+	ID string `json:"-"`
 
 	// GrantScope sets the scope the user authorized the client to use. Should be a subset of `requested_scope`.
-	GrantedScope sqlxx.StringSlicePipeDelimiter `json:"grant_scope" db:"granted_scope"`
+	GrantedScope sqlxx.StringSlicePipeDelimiter `json:"grant_scope"`
 
 	// GrantedAudience sets the audience the user authorized the client to use. Should be a subset of `requested_access_token_audience`.
-	GrantedAudience sqlxx.StringSlicePipeDelimiter `json:"grant_access_token_audience" db:"granted_at_audience"`
+	GrantedAudience sqlxx.StringSlicePipeDelimiter `json:"grant_access_token_audience"`
 
 	// Session allows you to set (optional) session data for access and ID tokens.
-	Session *ConsentRequestSessionData `json:"session" db:"-"`
+	Session *ConsentRequestSessionData `json:"session" faker:"-"`
 
 	// Remember, if set to true, tells ORY Hydra to remember this consent authorization and reuse it if the same
 	// client asks the same user for the same, or a subset of, scope.
-	Remember bool `json:"remember" db:"remember"`
+	Remember bool `json:"remember"`
 
 	// RememberFor sets how long the consent authorization should be remembered for in seconds. If set to `0`, the
 	// authorization will be remembered indefinitely.
-	RememberFor int `json:"remember_for" db:"remember_for"`
+	RememberFor int `json:"remember_for"`
 
 	// HandledAt contains the timestamp the consent request was handled.
-	HandledAt sqlxx.NullTime `json:"handled_at" db:"handled_at"`
+	HandledAt sqlxx.NullTime `json:"handled_at"`
 
 	// If set to true means that the request was already handled. This
 	// can happen on form double-submit or other errors. If this is set
 	// we recommend redirecting the user to `request_url` to re-initiate
 	// the flow.
-	WasHandled bool `json:"-" db:"was_used"`
+	WasHandled bool `json:"-"`
 
-	ConsentRequest  *ConsentRequest     `json:"-" db:"-"`
-	Error           *RequestDeniedError `json:"-" db:"error"`
-	RequestedAt     time.Time           `json:"-" db:"requested_at"`
-	AuthenticatedAt sqlxx.NullTime      `json:"-" db:"authenticated_at"`
-
-	SessionIDToken     sqlxx.MapStringInterface `db:"session_id_token" json:"-"`
-	SessionAccessToken sqlxx.MapStringInterface `db:"session_access_token" json:"-"`
-}
+	ConsentRequest  *ConsentRequest     `json:"-"`
+	Error           *RequestDeniedError `json:"-"`
+	RequestedAt     time.Time           `json:"-"`
+	AuthenticatedAt sqlxx.NullTime      `json:"-"`
 
-func (_ HandledConsentRequest) TableName() string {
-	return "hydra_oauth2_consent_request_handled"
+	SessionIDToken     sqlxx.MapStringInterface `json:"-" faker:"-"`
+	SessionAccessToken sqlxx.MapStringInterface `json:"-" faker:"-"`
```

Comment:
These two are not properly checked in `flow/flow_test.go` because faker sets them to zero.

Reply:
I added them manually to TestFlow_GetHandledConsentRequest manually